Gemfile

```diff
@@ -48,7 +48,7 @@ gem 'rdiscount', platforms: [:ruby]
 gem 'rdoc'
 gem 'redcarpet', platforms: [:ruby]
 gem 'simplecov', require: false
-gem 'slim', '~> 4'
+gem 'slim', '~> 5'
```

Comment:
TIL https://github.com/slim-template/slim/issues/851

Reply:
Yeah, I had to upgrade otherwise it was blocking upgrading `tilt` to `2.2.0` which is needed for this.